index.html

```diff
@@ -217,6 +217,20 @@ <h2 class="fz20px">Skills</h2>
     </article>
   </main>
   <footer>
+    <article class="bg-footer border-tl">
+      <section class="footer_class p5vh">
+        <h2 class="title">Contact me</h2>
+        <p class="details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="flex flex-column" action="https://formspree.io/f/mlevqnqq" method="post">
+          <input type="text" id="name-input" name="name" placeholder="Your name" maxlength="30" required>
+          <input type="text" id="email" name="email" placeholder="Your email" required>
+          <textarea name="" id="" cols="30" rows="10" placeholder="Your message" required maxlength="500"></textarea>
+          <div>
+            <button class="button-white">Get It Touch</button>
```

Comment:
- [ ] As this button will be the one that sends data to the form, kindly add a type of `submit`. [Button type ](https://www.w3schools.com/tags/att_button_type.asp)

Reply:
Thank you!